app/models/user.rb

```diff
@@ -9,7 +9,7 @@ class User < ApplicationRecord
            dependent: :destroy
   has_many :notifications_as_mentioned, class_name: 'ActivityNotification', dependent: :destroy
   has_many :messages, as: :sender, dependent: :destroy
-  belongs_to :organization, optional: true
+  acts_as_tenant :organization, optional: true
```

Comment:
I would not scope the user but allow him to habtm organizations. There are use cases where the user will manage many orgs

Reply:
We discussed this and I'll look to implement it